### ISSUE TITLE
Set initial trajectory position from coordinates for ship position

### ISSUE THREAD

**Issue by tdjames1** [cemac/mosaic_traj#2]
> the valid trajectory points are those where the pressure is positive values. In fact the trajectory is actually released at some point between the last negative and first positive point at the exact time and location of the ship position point in question. These positions are available in a file for each day:
>
> /nfs/see-fs-02_users/lecsra/MOSAiC/DRIFT/trelease

Do we want to plot the initial trajectory point from the "exact time and location of the ship position"?

**Comment by sarnoldlds** [comment]
Propose solution where trajectory data is interpolated such that it is spaced regularly at 3-hourly intervals at times before arrival at the exact time (and place) of release from the ship. 

In practice this means interpolating between real release time (from rsh files) and data with positive pressure points from the trajectory file. See IDL code where I have done this.

Interpolate using log of pressure and specific humidity.

**Comment by tdjames1** [comment]
See MOSAiC_trajs_actual.pro

Relevant routines are:
 INTERP_TO_RELEASE - Interpolates the raw trajectory output to time points that are 3 hours back from actual release time.
 READ_TRACKS - Reads in the rsh release files.
 READ_MATCH - Matches the offset filenames of the release files and the trajectory output files.

**Comment by tdjames1** [comment]
@sarnoldlds Are the diagrams that you drew during the meeting still on your whiteboard? I should have made a record of them at the time for reference!

**Comment by sarnoldlds** [comment]
> @sarnoldlds Are the diagrams that you drew during the meeting still on your whiteboard? I should have made a record of them at the time for reference!

Unfortunately not! Let me know if you want me to explain something again. I think it was to do with the interpolation to allow release from the exact ship time and 3-hourly backward steps?

**Comment by tdjames1** [comment]
Yes, I was trying to understand your interpolation code and I thought the diagram might help. Hopefully I can work it out - I'll let you know when I've had another look at it.
